server/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig to read more about this file */
+
+    /* Projects */
+    // "incremental": true,                              /* Save .tsbuildinfo files to allow for incremental compilation of projects. */
```

Comment:
Having trouble testing the config. Getting a white screen at the moment. Please add instructions on how to run the server and client including what to do when some situations arise.